typedoc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://typedoc.org/schema.json",
+  "out": "./docs/reference",
+  "entryPoints": [
+    "./src/index.ts"
```

Comment:
Could make the entrypoint more granular and then it would generate more than 1 file